suites/apis/etlTest.js

```diff
@@ -31,11 +42,14 @@ Scenario('run ETL second time @etl', async ({ I, sheepdog }) => {
 });
 
 AfterSuite(async ({ etl }) => {
-  etl.props.aliases.forEach((alias) => {
+  console.log("cleaning up indices created by ETL runs");
```

Comment:
can we not just call cleanUp method here?

Reply:
I really wanted to use `cleanUpIndices` here, but `cleanUpIndices` doesn't ask `hasVersionIncreased`.